frontend/frontend/honeylogger/_console.py

```diff
@@ -0,0 +1,60 @@
+import hashlib
+import logging
+import sys
+from ipaddress import IPv4Address, IPv6Address
+from typing import Optional, Union
+
+from ._types import IPAddress
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter(
+    fmt='%(asctime)s %(levelname)-8s %(message)s', datefmt='%Y-%m-%d %H:%M:%S')
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+
+class ConsoleLogSSHSession():
```

Comment:
If you plan to use this with a "with" statement you may want to define the __enter__ and __exit__ methods.